tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1227,18 +1226,21 @@ a dictionary can also be used with a named enum.
 
 ::
 
-    enum State {STATE_IDLE, STATE_JUMP = 5, STATE_SHOOT}
-
+    enum States {STATE_IDLE, STATE_JUMP = 5, STATE_SHOOT}
```

Comment:
Restore the name and the empty line, it matters for readability, and it makes more sense for this to be singular